### PR TITLE
Part12 spanish translation

### PR DESCRIPTION
**Pull request by manglynho**
Spanish Translation for part 12.

### CODE REVIEW

File: src/content/12/es/part12a.md [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+
+Por lo que los contenedores son utilizados en ambientes en la nube e incluso durante el desarrollo. Cuáles son los beneficios de utilizar contenedores? He aquí dos escenarios comunes:
+
+<i>Escenario 1: Estas desarrollando una aplicación nueva que necesita ejecutarse en la misma maquina que una aplicación antigua (legacy). Ambas requieren diferentes versiones de Node instalada.</i>
+
+Tu probablemente utilices nvm, maquinas virtuales o magia negra para lograr ejecutarlas al mismo tiempo. Sin embargo los contenedores son una excelente solucion ya que puedes ejecutar ambas aplicaciones en sus respectivos contenedores. Ellas estan aisladas una de otra y no interfieren.
+
+<i>Scenario 2: Tu aplicación se ejecuta en tu ordenador. Necesitas mover la aplicación a un servidor</i>
```
Comment (RamonEspinosa) [inline src/content/12/es/part12a.md:26]: Here's a small typo. The correct word should be "Escenario"

File: src/content/12/es/part12a.md [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+No es poco común que la aplicacion simplemente no se ejecute en el servidor a pesar de estar trabajando bien en tu computadora. Esto puede ocurrir debido a algunas dependencias faltantes o otras diferencias en los entornos. Aquí los contenedores son una excelente solución ya que puedes ejecutar tu aplicación en el mismo ambiente tanto en tu computadora como en el servidor. No es perfecto: las diferencias en el hardware pueden provocar incidentes, pero puedes limitar estas diferencias entre los ambientes.
+
+Alguna ves podrás escuchar sobre el tema <i>"Works in my container"</i>. La frase describe la situación en la que la aplicación funciona bien en un contenedor ejecutándose en tu computadora pero se rompe cuando el contenedor es iniciado en el servidor. Esta frase es una variante del infame tema <i>"Works on my machine"</i>, que con frecuencia resuelven los contenedores. La situación es también con mucha certeza un error de uso.
```
Comment (RamonEspinosa) [inline src/content/12/es/part12a.md:30]: "Alguna ves" should be "Alguna vez" instead.

File: src/content/12/es/part12a.md [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+
+Si solo ha utilizado una interfaz gráfica de usuario y nunca ha tocado, p. Linux o terminal en Mac, o si te quedas atascado en los primeros ejercicios, te recomendamos hacer primero la Parte 1 de "Herramientas informáticas para estudios de CS": <https://tkt-lapio.github.io/en/>. Omita la sección "Conexión SSH" y el Ejercicio 11. ¡Esto incluye todo lo que necesitará para comenzar aquí!
+
+#### Ejercicio 12.1: Usando una computadora (sin la interfaz gráfica de usuario)
+
+Paso 1: Lee el texto debajo de la cabecera Advertencia.
+
+Paso 2: Descargue este [repositorio](https://github.com/fullstack-hy2020/part12-containers-applications) y conviértelo en tu repositorio de envío de esta parte del curso.
+
+Paso 3: Ejecuta <i>curl http://helsinki.fi</i> y guarda el resultado en un archivo. Guarda el archivo en tu repositorio con el nombre <i>script-answers/exercise12_1.txt</i>. El directorio <i>script-answers</i> ha sido creado en el paso anterio.
```
Comment (RamonEspinosa) [inline src/content/12/es/part12a.md:55]: There's a missing "r" in the word "anterior".

File: src/content/12/es/part12a.md [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+
+Para administrar los contenedores docker, también existe una herramienta llamada [Docker Compose](https://docs.docker.com/compose/) que permite **orquestar** (controlar) varios contenedores al mismo tiempo. En esta parte, utilizaremos Docker Compose para configurar un entorno de desarrollo local complejo. En la versión final del entorno de desarrollo que configuramos, incluso instalar Node en nuestra máquina ya no es un requisito.
+
+Hay varios conceptos que necesitamos repasar. ¡Pero los omitiremos por ahora y aprenderemos sobre Docker primero!
+
+Comencemos con el comando <i>docker container run</i> que se usa para ejecutar imágenes dentro de un contenedor. La estructura del comando es la siguiente: _container run <i>IMAGE-NAME</i>_ le indicaremos a Docker que cree un contenedor a partir de una imagen. Una característica particularmente interesante del comando es que puede ejecutar un contenedor incluso si la imagen para ejecutar aún no se ha descargado en nuestro dispositivo.
+
+Ejecutemos el comandod
```
Comment (RamonEspinosa) [inline src/content/12/es/part12a.md:124]: Please remove the last "d" from the word "comando"

**Review by RamonEspinosa — CHANGES_REQUESTED** [review]
Thanks for translating! I found a couple of typos that could be easily solved before publishing this translation 😄

**Comment by Kaltsoon** [comment]
Great stuff! I fixed those typos pointed out by @RamonEspinosa. Thank you both!

**Comment by manglynho** [comment]
Thank you both for the help and the fast response. I'm very happy to help.
Best regards.

**Pull request merged**